common/buildings/babylon_military.txt

```diff
@@ -1,26 +1,24 @@
-﻿
-building_naval_command = {
+﻿building_naval_command = {
 	building_group = bg_command
 	texture = "gfx/interface/icons/building_icons/naval_base.dds"
 	city_type = city
 	has_max_level = yes
 	levels_per_mesh = 0
-
-	can_build_government = {
+	possible = {
+		is_capital = yes
 		error_check = {
+			severity = potential
 			is_capital = yes
 		}
 	}
```

Comment:
does this mean that private construction can build it?

Reply:
Fixed :)